### DISCUSSION TITLE
Welcome to Genshin_StarRail_fps_unlocker Discussions!

### DISCUSSION THREAD

**Discussion by winTEuser** [winTEuser/Genshin_StarRail_fps_unlocker#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Herenxiaode** [comment]
第一次用,有个小问题.正常启动原神失去焦点后依旧维持60帧,使用解帧120后失去焦点就会降到10帧出现明显卡顿,在多屏下感觉不是很理想.失去焦点后哪怕是恢复60帧也行,降到10帧有点明显了...显卡占用不高.

**Reply by winTEuser** [reply]
配置文件里有个PowerSaveTarget，自己改
